app/models/quiz.rb

```diff
@@ -27,6 +27,10 @@ class Quiz < ApplicationRecord
 
   before_validation :set_time_limit_in_seconds
 
+  scope :featured, -> {
+    where(active: true).where('featured_at is not null').select { |quiz| Time.current > quiz.featured_at && Time.current < quiz.featured_at + 1.day.after }
```

Comment:
where.not(featured_at: nil)
Please refacftor - avoid ruby in scopes
Use query interface.
Create a scope for active